cocos/base/CCEventMouse.h

```diff
@@ -189,6 +196,8 @@ class CC_DLL EventMouse : public Event
     friend class EventListenerMouse;
 };
 
+std::ostream& operator<<(std::ostream& out, const EventMouse::MouseButton& button);
```

Comment:
Yep, easy printing is what it's there for. It means that the enum value can be written to ``std::ostream`` objects.

e.g.
```cpp
std::cout << mouseButton;
std::ostringstream << mouseButton;
```
``std::ostream`` is the standard mechanism for input/output in C++.
It's a convenience so it doesn't have to be included in the changes.